src/components/SarManageWidget/AddStake/index.tsx

```diff
@@ -113,6 +113,15 @@ export default function AddStake({ selectedOption, selectedPosition, onChange, o
     isLoading: isLoadingAssociate,
   } = useHederaTokenAssociated(sarNftContract?.address, 'Pangolin Sar NFT');
 
+  const fiatNumberFormmater = new Intl.NumberFormat(undefined, {
+    notation: 'compact',
+    style: 'currency',
+    currency: 'USD',
+    currencySign: 'accounting',
+    currencyDisplay: 'narrowSymbol',
+    maximumFractionDigits: 2,
+  });
```

Comment:
lets use `numeral` as we have been using that at multiple places to be consistent.